scripts/main.js

```diff
@@ -1 +1,117 @@
-// TODO: add JS
\ No newline at end of file
+// ------* selecting elements that will change with scene selection *--------
+const html = document.documentElement;
+const header = document.querySelector('h1');
+const select = document.querySelector('select');
+const audio = document.querySelector('audio');
+
+// ------* function declarations *---------
+
+function onSceneOptionSelected(event) {
+    const sceneSelection = event.target.value;
+    localStorage.setItem('selection', sceneSelection);
```

Comment:
Note: you are not validating your strings before setting to local storage. I do suggest exiting out of this function if the string does not equal one of the expected strings, otherwise you are opening the door to some hard to find bugs. This is a good place to think defensively